pkg/workloads/console/controller.go

```diff
@@ -213,6 +218,32 @@ func (r *reconciler) createOrUpdate(expected reconcile.ObjWithMeta, kind string,
 	return nil
 }
 
+// Ensure the console timeout is between [1, template.MaxTimeoutSeconds]
```

Comment:
This isn't *entirely* true. `template.Spec.DefaultTimeoutSeconds` can be 0; in which case we're actually setting the timeout to 0.

Perhaps it's worth changing the validation on this field to be a minimum of 1?

Reply:
But maybe setting the min to 1 would break some assumptions we have in the tests...

Reply:
I'd rather just update this comment - I think it's unlikely anyone is going to care whether the timeout is 0 or 1.